_topic_map.yml

```diff
@@ -1088,13 +1088,42 @@ Topics:
     File: planning-migration-3-to-4
   - Name: Migrating OpenShift Container Platform 3.7 to 4.2
     File: migrating-openshift-3-to-4
-- Name: Migrating OpenShift Container Platform 4.x to 4.x
-  Dir: migrating-4-4
+  - Name: Configuring a replication repository
+    File: configuring-replication-repository
+  - Name: Deploying the Cluster Application Migration tool
+    File: deploying_cam
+  - Name: Migrating applications with the CAM web console
+    File: migrating-applications-with-cam
+  - Name: Migrating control plane settings with the Control Plane Migration Assistant
+    File: migrating-with-cpma
+  - Name: Troubleshooting
+    File: troubleshooting
+- Name: Migrating OpenShift Container Platform 4.1 to 4.2
+  Dir: migrating-4_1-4
   Topics:
   - Name: Migrating OpenShift Container Platform 4.1 to 4.2
```

Comment:
I'm not sure that's a good idea. The "About" section was added by Andrea and is independent. The "Migrating ... 4.1 to 4.2" is nearly identical to "Migrating ... 3 to 4". I don't want to rename a module when its content is the same as another's, because that will make it much more difficult to identify and update. Possibly, the "About" section should be at a higher level. 

The "Migrating ..." assemblies were originally written as standalone assemblies. I'll think about renaming it but not with "About", because its content is very different from the existing "About" module.

Reply:
I see that the "About" section has changed and that Andrea refers to the "Migrating ... 3 to 4" section as "Performing the migration". That might work.